src/04-modality-specific-files/10-functional-near-infrared-spectroscopy.md

```diff
@@ -0,0 +1,367 @@
+# Functional Near-Infrared Spectroscopy
+
+Support for functional Near-Infrared Spectroscopy (fNIRS) was developed as a
+[BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals).
+Please see [Citing BIDS](../01-introduction.md#citing-bids)
+on how to appropriately credit this extension when referring to it in the
+context of the academic literature.
+
+## fNIRS recording data
+
+{{ MACROS___make_filename_template(datatypes=["nirs"], suffixes=["nirs", "events"]) }}
+
+Only the Shared Near Infrared File Format ([SNIRF](https://github.com/fNIRS/snirf))
+format is supported in BIDS. The SNIRF
+specification supports one or more fNIRS datasets to be stored in a single
+`.snirf` file. However, to be BIDS compatible each SNIRF file MUST contain
+only a single run. A limited set of fields from the SNIRF specification are
+replicated  in the BIDS specification. This redundancy allows the data to be
+easily parsed by humans and machines that do not have a SNIRF reader at hand,
+which improves findability and tooling development.
+
+Original unprocessed fNIRS data in the native format, if different, can also
+be stored in the [`/sourcedata` directory](../02-common-principles.md#source-vs-raw-vs-derived-data)
+along with code to convert the data to
+SNIRF in the [`/code` directory](../02-common-principles.md#storage-of-derived-datasets).
+The unprocessed raw data should be stored in
+the manufacturers format before any additional processing or conversion.
+The native file format is especially valuable in case conversion elicits the
+loss of crucial metadata specific to manufacturers and specific fNIRS systems.
+We also encourage users to provide additional meta information extracted from
+the manufacturer specific data files in the sidecar JSON file. This allows for
+easy searching and indexing of key metadata elements without needing to parse
+the various proprietary (typically binary) native data files. Other relevant
+files should  be included alongside the fNIRS data.
+
+### Terminology
+
+For proper documentation of fNIRS recording metadata it is important
+to understand the difference between a source, detector, and channel as these
+are defined differently to other modalities such as EEG. The following definitions
+apply in this document:
+
+-   Source - A light emitting device, sometimes called a transmitter.
+
+-   Detector - A photoelectric transducer, sometimes called a receiver.
+
+-   Channel - A paired coupling of a source and a detector with one specific light wavelength.
+    It is common for a single Source-Detector pair to result in two or more channels
+    with different wavelengths.
```

Comment:
Do you also want to define "Optode" here as well? You use the term a lot later on in the document. "Optode can refer to either a source or a detector" etc etc

Reply:
This terminology section is very helpful, thanks!!